TwitchPlaysAssembly/Src/Commands/GameCommands.cs

```diff
@@ -630,12 +641,22 @@ public static void CallAllQueuedCommands(string user, bool isWhisper)
 		var allCommands = TwitchGame.Instance.CommandQueue.ToList();
 		TwitchGame.Instance.CommandQueue.Clear();
 		TwitchGame.ModuleCameras?.SetNotes();
+		List<IRCMessage> cmdsToExecute = new List<IRCMessage>();
 		foreach (var call in allCommands)
 		{
 			IRCConnection.SendMessageFormat("Calling {0}: {1}", call.Message.UserNickName, call.Message.Text);
-			IRCConnection.ReceiveMessage(call.Message);
+			if (TwitchGame.Instance.Bombs.Any(x => x.BackdoorHandleHack))
+				IRCConnection.SendMessage("Hack detected, waiting until the hack is over to execute this command.");
+			cmdsToExecute.Add(call.Message);
 		}
 		DeleteCallInformation(true);
+		if (calledCommands.Count == 0)
+		{
+			calledCommands.AddRange(cmdsToExecute);
+			TwitchGame.Instance.StartCoroutine(WaitForCall());
+		}
+		else
+			calledCommands.AddRange(cmdsToExecute);
```

Comment:
I think it would be better to pass `cmdsToExecute` to WaitForCall. WaitForCall could then handle this logic. Something like:
```csharp
bool alreadyExecuting = calledCommands.Count != 0;
calledCommands.AddRange(cmdsToExecute);
if (alreadyExecuting) yield break;
```